src/main/java/com/rabbitmq/http/client/domain/ParameterWrapper.java

```diff
@@ -0,0 +1,57 @@
+package com.rabbitmq.http.client.domain;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class ParameterWrapper<T> {
```

Comment:
Can this be `RuntimeParameter` instead? I'm not sure what is being wrapped here since this is a POJO?

Reply:
Renamed to `RuntimeParameter`